backend/src/routes/user.ts

```diff
@@ -21,17 +23,48 @@ router.get('/:username', async (ctx, next) => {
 	ctx.body = user;
 });
 
-router.put('/:username', async (ctx, next) => {
+//Create a new user with required handle and display name
+router.put('/newuser/:handle/:displayname', async (ctx, next) => {
```

Comment:
it's more conventional to have just the handle/username in the route params and to move displayname and other user info to the request body (`ctx.body.displayname` for example), since the username is what uniquely identifies the object!

Reply:
also, let's just leave the route to "/:username" instead of "/newuser/:username", because then it doubles as a route to edit a user